tests/zip/zip_test.py

```diff
@@ -26,6 +26,7 @@
 # Unix dir bit and Windows dir bit. Magic from zip spec
 UNIX_DIR_BIT = 0o40000
 MSDOS_DIR_BIT = 0x10
+UNIX_RWX_BITS = 0o777
```

Comment:
Use symbolic constants from the `stat` module instead?

Reply:
I'm inclined to keep the same style as the other magic bits.